File: infoclinCore/src/ifrn/pds/model/Usuario.java
package ifrn.pds.model;

import java.io.Serializable;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "Usuario")
public class Usuario implements Serializable {

	private static final long serialVersionUID = 1L;
	@Id
	@GeneratedValue(strategy = GenerationType.SEQUENCE)
	private int id;
	private String login;
	private String senha;
	private String nome;
	private int cpf;
	private Endereco endereco;

	public Usuario() {
	}

	public Usuario(String senha, String login, Endereco endereco,
			String nome, int cpf) {
		this.senha = senha;
		this.login = login;
		this.endereco = endereco;
		this.nome = nome;
		this.cpf = cpf;
	}

	public void setSenha(String senha) {
		this.senha = senha;
	}

	public String getSenha() {
		return senha;
	}

	public void setLogin(String login) {
		this.login = login;
	}

	public String getLogin() {
		return login;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getNome() {
		return nome;
	}


	public void setEndereco(Endereco endereco) {
		this.endereco = endereco;
	}

	public Endereco getEndereco() {
		return endereco;
	}

	public void setId(int id) {
		this.id = id;
	}

	public int getId() {
		return id;
	}

	public int getCpf() {
		return cpf;
	}

	public void setCpf(int cpf) {
		this.cpf = cpf;
	}

}

File: trunk/infoclinDesktop/InfoClin/Dominio/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Dominio
{
    public class Usuario
    {
        //int Id; vem do web service
        private string login;
        private string senha;
        private Endereco endereco { get; set; }
        private string nome;
        private int cpf;
        
        public Usuario(/*int id,*/ string login, string senha, Endereco endereco,
            string nome, int cpf)
        {
            //Id = id;
            this.login = login;
            this.senha = senha;
            this.endereco = endereco;
            this.nome = nome;
            this.cpf = cpf;
        }

        public string Nome
        {
            get { return this.nome; }
            set { this.nome = value; }
        }

        

        public void conectar()
        {

        }

        
    }
}
